<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-PER5DOON" ignored="1" name="Personal Loan" tests="21" failures="1" timestamp="2020-06-15T00:40:05 BST" time="0.075" errors="0">
  <testcase name="firstTestNG" time="0.002" classname="TestNGBasic.Step1TestNG"/>
  <testcase name="apiHomeloan" time="0.001" classname="TestNGBasic.Step4HomeLoan"/>
  <testcase name="ploan" time="0.001" classname="TestNGBasic.Step2Loan"/>
  <testcase name="setData" time="0.002" classname="TestNGBasic.Step6DataProvider"/>
  <testcase name="webCarLoan" time="0.002" classname="TestNGBasic.Step3CarLoan"/>
  <testcase name="facebookLogin" time="0.005" classname="TestNGBasic.Step5MobileTest"/>
  <testcase name="setData" time="0.001" classname="TestNGBasic.Step6DataProvider"/>
  <testcase name="mobileCarLoan" time="0.0" classname="TestNGBasic.Step3CarLoan"/>
  <testcase name="secondTest" time="0.002" classname="TestNGBasic.Step1TestNG"/>
  <testcase name="setData" time="0.0" classname="TestNGBasic.Step6DataProvider"/>
  <testcase name="instaLogin" time="0.0" classname="TestNGBasic.Step5MobileTest"/>
  <testcase name="apiCarloan" time="0.002" classname="TestNGBasic.Step3CarLoan"/>
  <testcase name="mobileHomeLoan" time="0.004" classname="TestNGBasic.Step4HomeLoan">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at TestNGBasic.Step4HomeLoan.mobileHomeLoan(Step4HomeLoan.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- mobileHomeLoan -->
  <testcase name="mobileApp" time="0.001" classname="TestNGBasic.Step5MobileTest"/>
  <testcase name="webHomeLoan" time="0.0" classname="TestNGBasic.Step4HomeLoan"/>
  <testcase name="mobileCamera" time="0.0" classname="TestNGBasic.Step5MobileTest"/>
  <testcase name="mobileFilemanager" time="0.0" classname="TestNGBasic.Step5MobileTest"/>
  <testcase name="mobileGallery" time="0.0" classname="TestNGBasic.Step5MobileTest"/>
  <testcase name="mobileLogin" time="0.0" classname="TestNGBasic.Step5MobileTest"/>
  <testcase name="mobileName" time="0.0" classname="TestNGBasic.Step5MobileTest"/>
  <testcase name="mobileRegister" time="0.0" classname="TestNGBasic.Step5MobileTest"/>
  <testcase name="whattsappLogin" time="0.0" classname="TestNGBasic.Step5MobileTest">
    <ignored/>
  </testcase> <!-- whattsappLogin -->
</testsuite> <!-- Personal Loan -->
